projects/robots/boston_dynamics/spot/controllers/moving_demo/moving_demo.c

```diff
@@ -0,0 +1,299 @@
+/*
```

Comment:
This name looks too generic and might cause conflicts with other controller, I would prefix the ``spot`` keyword.

P.S: I am not reviewing the content of the file as controllers may not be ready for now.